object/application.go

```diff
@@ -103,6 +103,16 @@ func GetApplications(owner string) ([]*Application, error) {
 	return applications, nil
 }
 
+func CountApplicatoinsByProvider(providerName string) ([]*Application, error) {
```

Comment:
can we use 
api/get-applications?owner=?&field=providers&value=providerName instad?